.eslintrc.js

```diff
@@ -25,7 +25,17 @@ module.exports = {
       rules. For now, warn on these.
      */
     "import/no-relative-parent-imports": "warn",
-    "no-restricted-imports": "warn",
+    "no-restricted-imports": [
+      "error",
+      {
+        paths: [
+          {
+            name: "lodash",
+            message: "Import [module] from lodash/[module] instead",
```

Comment:
Minor suggestion: add a period on the end to make it more readable
```suggestion
            message: "Import [module] from lodash/[module] instead.",
```